src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,54 @@
  */
 
 public class Plane {
+    private String model;
+    private int speed;
+    private int weight;
+    private int age;
+    private String color;
 
+    public Plane(String model, int speed, int weight, int age, String color) {
+        this.model = model;
+        this.speed = speed;
+        this.weight = weight;
+        this.age = age;
+        this.color = color;
+    }
+
```

Comment:
for correct Builder pattern, we need private constructor of Plane with a Builder object

Reply:
I`ve fixed that too.